capstone/src/main/java/servlet/Event.java

```diff
@@ -0,0 +1,28 @@
+public class Event{
```

Comment:
Does this need setters as well? Or do you anticipate only setting the fields when you first create the object?

Reply:
Silas said he needed only getters